helm/kanister-operator/values.yaml

```diff
@@ -24,6 +24,10 @@ controller:
   updateCRDs: true
 bpValidatingWebhook:
   enabled: true
+  tls:
+    mode: auto #<If set to `custom` then secretName and caBundle should be provided>
+    secretName: '' # Pre created Secret in application namespace having tls certs details
```

Comment:
what is application namespace? is this the namespace where kanister is being installed?

Reply:
yes, Kanister application.